src/globus_sdk/services/search/client.py

```diff
@@ -237,10 +245,25 @@ def scroll(
         :param marker: marker used in paging (overwrites any marker in ``data``)
         :type marker: str, optional
 
-        **Examples**
+        .. tab-set::
 
-        >>> sc = globus_sdk.SearchClient(...)
-        >>> scroll_result = sc.scroll(index_id, {"q": "*"})
+            .. tab-item:: Example Usage
+
+                .. code-block:: pycon
+
+                    >>> sc = globus_sdk.SearchClient(...)
+                    ... scroll_result = sc.scroll(index_id, {"q": "*"})
```

Comment:
```suggestion
                    >>> scroll_result = sc.scroll(index_id, {"q": "*"})
```

Reply:
Thanks for this batch of catches on the pycon blocks!
blacken-docs was unhappy with everything being `>>>`, and I started fixing them up with some find-replace for `>>>` to `...`. But then I realized that I had gone too far and went _back_ to fix several cases back to `>>>`, and clearly I missed a couple.